pages/developers/guidelines.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Package requirements and guidelines
+permalink: /guidelines
+nav_order: 1
+parent: Developer information
+---
+
+# Package requirements and guidelines
+
+The below provides requirements and guidelines for Scikit-HEP packages,
+which we hope help providing high-quality packages with some level of homogeneity across them.
+
+### Package license
+
+Every package is required to have a license.
+As a  guideline, we try to be as open as possible with licensing,
+following what most of the other packages use in the wide Python scientific community;
+Scikit-HEP packages typically use the BSD-3 license.
+It is acknowledged that exceptions are needed and "sticky licenses" such as GPL may be the only viable way.
+
+### README
+
+A `README.[md,rst]` file is required.
+It should provide an overview of the package and at least a quick getting started section.
+There are several examples on how to go about this in the various Scikit-HEP packages.
+
+### Documentation
+
+Documentation in the code is a must and so-called docstrings should permeate the code as much as possible.
+It is suggested to provide documentation via [ReadTheDocs][].
+
+### Code formatting.
+
+We suggest as a guideline to use [Black][], which is rather popular these days.
+See [/style][] for details.
```

Comment:
This is most probably the wrong way to refer to the same-level page https://scikit-hep.org/developer/style. I wrote a guess, I confess. Please amend if needed ...

Reply:
I think there's a filter you should use, but I have to first see which version of Jekyll we use - this was simplified in 4.0.

Reply:
OK, sounds good.

Reply:
@henryiii, any idea, then? You probably use such a feature in the IRIS-HEP pages?

Reply:
I’ll check in ~5 mins

Reply:
Looks like GH pages is still stuck on 3.x. I'll update these with the 3.x syntax.